Crd config restructure (#52)
banzaicloud/logging-operator-docs: docs/headless/component-overview.md

> You can define `outputs` (destinations where you want to send your log messages, for example, Elasticsearch, or an Amazon S3 bucket), and `flows` that use filters and selectors to route log messages to the appropriate outputs. You can also define cluster-wide outputs and flows, for example, to use a centralized output that namespaced users cannot modify.

Reviewer: ```suggestion
You can define `outputs` (destinations where you want to send your log messages, for example, Elasticsearch, or an Amazon S3 bucket), and `flows` that use filters and selectors to route log messages to the appropriate outputs. You can also define cluster-wide outputs and flows, for example, to use a centralized output that namespaced users can reference but cannot modify.
```

> You can configure the Logging operator using the following Custom Resource Descriptions.

Reviewer: ```suggestion
You can configure the Logging operator using the following Custom Resource Definitions.
```

- [logging]({{< relref "docs/one-eye/logging-operator/configuration/logging.md" >}}) - The `logging` resource defines the logging infrastructure for your cluster that collects and transports your log messages. It also contains configurations for Fluentd and Fluent-bit.
- [output]({{< relref "docs/one-eye/logging-operator/configuration/output.md" >}}) - Defines an Output for a logging flow, where the log messages are sent. This is a namespaced resource. See also `clusteroutput`.
- [flow]({{< relref "docs/one-eye/logging-operator/configuration/flow.md" >}}) - Defines a logging flow using `filters` and `outputs`. Basically, the flow routes the selected log messages to the specified outputs. This is a namespaced resource. See also `clusterflow`.

> - [clusteroutput]({{< relref "docs/one-eye/logging-operator/configuration/output.md" >}}) - Defines an output without namespace restriction. Only effective in `controlNamespace`.
> - [clusterflow]({{< relref "docs/one-eye/logging-operator/configuration/flow.md" >}}) - Defines a logging flow without namespace restriction.

Reviewer: ```suggestion
- [clusteroutput]({{< relref "docs/one-eye/logging-operator/configuration/output.md" >}}) - Defines an output that is available from all flows and clusterflows. The operator evaluates clusteroutputs in the `controlNamespace` only unless `allowClusterResourcesFromAllNamespaces` is set to true.
- [clusterflow]({{< relref "docs/one-eye/logging-operator/configuration/flow.md" >}}) - Defines a logging flow that collects logs from all namespaces by default. The operator evaluates clusterflows in the `controlNamespace` only unless `allowClusterResourcesFromAllNamespaces` is set to true.
```
